docs/how-to/migration.rst

```diff
@@ -0,0 +1,118 @@
+=======================
+Service Control
+=======================
+
+Service control refers to the set of commands for enabling and disabling a
+given MicroOVN service. MicroOVN has a set of services referred to here
+:doc:`Services Reference </reference/services>`, which are responsible for
+handling core functionality.
+
+You can disable services manually using snap, but this does not update the
+database or handle properly joining clusters/configuring the service properly,
+hence the strong reasoning to interact with services through this method
+
+.. note::
+
+   This assumes you have MicroOVN installed and a MicroOVN cluster of at least 3
+   nodes set up. These nodes will be referred to as ``first``, ``second`` and
+   ``third`` respectively.
+
+Disabling a MicroOVN service
+----------------------------
+
+Disabling a MicroOVN service will configure it to not start automatically at
+boot and stop the service if it is running.
+
+run on ``first``:
+
+.. code-block:: none
+
+   microovn disable switch
+   Service switch disabled
+
+This will disable the switch service in MicroOVN, This can be shown through the
+listing of system services owned by MicroOVN. These do not always translate
+directly to a MicroOVN service, but in this case it does.
+
+run on ``first``:
+
+.. code-block:: none
+
+   snap services microovn | grep switch
+   microovn.switch                  disabled   inactive  -
+
+You should be able to see here that the service is not running and disabled on
+startup. The other nodes are also aware of this as if you query the status.
+
+run on ``second``:
+
+.. code-block:: none
+
+   microovn status
+   MicroOVN deployment summary:
+   - first (10.190.155.5)
+   Services: central, chassis
+   - second (10.190.155.174)
+   Services: central, chassis, switch
+   - third (10.190.155.55)
+   Services: central, chassis, switch
+   OVN Database summary:
+   OVN Northbound: OK (7.3.0)
+   OVN Southbound: OK (20.33.0)
+
+
+Enabling a MicroOVN service
+----------------------------
+
+Enabling a MicroOVN service will configure it to start automatically at boot and
+if the service is not running, start it.
+
+run on ``first``:
+
+.. code-block:: none
+
+   microovn enable switch
+   Service switch enabled
+
+.. warning::
+
+   If the switch service is enabled you may get an error, this is fine.
```

Comment:
I'd suggest to drop this warning, if you *really* want to keep it, downgrade it to a `note` and phrase it more generically, such as "If the service is already enabled you will get an error message."